app/screens/TodoList/layout.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View, StyleSheet, FlatList } from 'react-native';
 
-import TodoList from './components/TodoList';
+import TodoItemContainer from './components/TodoItem';
 import Header from './components/Header';
 import TodoInput from './components/TodoInput';
 import RemoveAllItems from './components/RemoveAllItems';
 
 function TodoListScreen({ onAddItem, listData, onDeleteItem, onRemoveAllItems }) {
+  renderList = item => (
+    <TodoItemContainer onDeleteItem={onDeleteItem} index={item.key} item={item} />
+  );
+
   return (
     <View style={styles.container}>
       <Header />
       <TodoInput onAddItem={onAddItem} />
-      <TodoList
-        listData={listData}
-        onDeleteItem={onDeleteItem}
+      <FlatList
+        style={styles.listContainer}
+        data={listData}
+        renderItem={({ item }) => renderList(item)}
```

Comment:
check that if `renderItem` is injecting an item to the arrow function, you can just use the function :
```
renderList = ({ item }) => (
    <TodoItemContainer onDeleteItem={onDeleteItem} index={item.key} item={item} />
  );
```

and using in the flatlist like:
`renderItem={renderList}`